flink-contrib/flink-storm/src/test/java/org/apache/flink/storm/wrappers/StormTupleTest.java

```diff
@@ -595,7 +593,7 @@ public void testGetBinaryByFieldPojoGetter() throws Exception {
 	private <T> StormTuple testGetByField(int arity, int index, T value)
 			throws Exception {
 
-		assert (index < arity);
+		Assert.assertTrue(index < arity);
 
```

Comment:
Is it? I am not aware if this. Why? Using `assert` to check internal code invariants is the best way to go IMHO.